### PR TITLE
Add stub for webarchive-seed viewer

### LINKED ISSUE #246: Stub out the initial web archive seed viewer class
This would be in ruby, and would allow web archive seed purls to be viewed as a web archive object

### PR DESCRIPTION
**Pull request by aalsum**
close #246

### CODE REVIEW

File: spec/javascripts/was_seed/was_seed_spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//= require jquery
+//= require was_seed
+
+'use strict';
+
+describe('WAS Seed Viewer', function() {
+  describe('loads and makes available modules', function() {
+    it('WasSeedViewer is defined', function() {
+      expect(WasSeedViewer).toBeDefined();
```
Comment (mejackreed) [inline spec/javascripts/was_seed/was_seed_spec.js]: This style violation can be avoided now by putting this at the top of the file:

```javascript
/*global WasSeedViewer */

//= require jquery....
```

File: app/assets/javascripts/was_seed.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+//= require modules/common_viewer_behavior
+//= require modules/popup_panels
+//= require modules/embed_this
+//= require modules/was_seed_viewer
+
+CssInjection.injectFontAwesome();
+CssInjection.appendToHead();
+CommonViewerBehavior.initializeViewer(WasSeedViewer.init);
+PopupPanels.init();
+EmbedThis.init();
```
Comment (mejackreed) [inline app/assets/javascripts/was_seed.js]: This style violation can be avoided now by putting this at the top of the file:

```javascript
/*global EmbedThis, CssInjection, ComonViewerBehavior, PopupPanels */

//= require jquery....
```

File: spec/features/was_seed_viewer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+include PURLFixtures
+
+describe 'was seed viewer public', js: true do
+  before do
+    stub_purl_response_with_fixture(was_seed_purl)
+    visit_sandbox
+    fill_in_default_sandbox_form
+    click_button 'Embed'
+  end
```
Comment (mejackreed) [inline spec/features/was_seed_viewer_spec.rb]: Should there be a test here? Maybe we should test that the thing shows up with a title?

**Comment by mejackreed** [comment]
Looks great. A couple minor style things that we now know how to fix. Also I was wondering if there should be a test in `was_seed_viewer_spec.rb`? Super minor, if you don't want to fix, feel free to tell me and I will happily merge.

File: spec/features/was_seed_viewer_spec.rb (hunk 3 above)
Comment (aalsum) [inline spec/features/was_seed_viewer_spec.rb]: It's a holder for the upcoming implementation and testing.

**Comment by aalsum** [comment]
I think it's ready for merging now.

**Pull request merged**